sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -127,6 +127,9 @@ final class ManifestMetadataReader {
   static final String ENABLE_AUTO_TRACE_ID_GENERATION =
       "io.sentry.traces.enable-auto-id-generation";
 
+  static final String AUTO_TRANSACTION_DEADLINE_TIMEOUT_MILLIS =
+      "io.sentry.traces.auto-transaction-deadline-timeout-millis";
```

Comment:
should we name it just `deadline-timeout`? Akin to `idle-timeout`. And same for the Java property if we change this one